app-benchmarks/unigine-superposition/unigine-superposition-1.1.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit desktop xdg-utils
+
+DESCRIPTION="Game like benchmark (CPU, GPU), for ~5 minutes"
+HOMEPAGE="https://benchmark.unigine.com/superposition"
+
+SRC_URI="https://assets.unigine.com/d/Unigine_Superposition-${PV}.run"
+S="${DISTDIR}"
+
+LICENSE="all-rights-reserved"
```

Comment:
This package is not all-rights-reserved, it has an EULA: https://benchmark.unigine.com/policies/superposition-eula

What you need to do is copy that EULA to `licenses` and use it as `LICENSE`.

EULAs can be even more restrictive than all-right-reserved.

Reply:
Thanks, done.